iota-client/core/get_node_info.rs

```diff
@@ -30,16 +30,16 @@ pub struct GetNodeInfoResponse {
     app_version: String,
     /// Number of threads IRI is using
     #[serde(rename = "jreAvailableProcessors")]
-    jre_available_processors: usize,
+    jre_available_processors: u64,
```

Comment:
this maximum processor number is crazy. maybe u16?

Reply:
sorry, I saw jre and just assigned it to u64 😂